var/spack/repos/builtin/packages/darshan-util/package.py

```diff
@@ -62,55 +65,15 @@ def configure_args(self):
         extra_args.append('CC=%s' % self.compiler.cc)
         extra_args.append('--with-zlib=%s' % spec['zlib'].prefix)
         if '+apmpi' in spec:
-            if self.version < Version('3.3.2'):
-                extra_args.append('--enable-autoperf-apmpi')
-            else:
                 extra_args.append('--enable-apmpi-mod')
         if '+apxc' in spec:
-            if self.version < Version('3.3.2'):
-                extra_args.append('--enable-autoperf-apxc')
-            else:
                 extra_args.append('--enable-apxc-mod')
 
+        if '+shared' in spec:
+            extra_args.append('--enable-shared')
+
         return extra_args
 
     @property
     def basepath(self):
         return join_path('darshan-test', 'example-output')
-
```

Comment:
Why remove the spack testing from this?

Reply:
Because it won't work with my other changes yet--feel free to push in the fixes, related to new variants I've added changing a path I think. Note that none of this is close to production ready yet.

I can get you the traceback it produces if that helps.